renovate.json5

```diff
@@ -0,0 +1,36 @@
+{
+  extends: [
+    "config:base",
+    ":disableRateLimiting",
+  ],
+  automerge: false,
+  platformAutomerge: false,
+  postUpdateOptions: ["gomodTidy", "yarnDedupeHighest"],
+  // https://docs.renovatebot.com/modules/manager/regex/
+  regexManagers: [
+    {
+      fileMatch: ["(^|/)Dockerfile$", "(^|/)Dockerfile\\.[^/]*$"],
+      matchStrings: [
+        // match `ENV *_VERSION=x.x.x` with renovate configurations
+        "renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)( versioning=(?<versioning>.*?))?\\sENV .*?_VERSION=(?<currentValue>.*)\\s",
+      ],
+      versioningTemplate: "{{#if versioning}}{{{versioning}}}{{else}}semver{{/if}}"
```

Comment:
Currently this setting find following comment.

```
# renovate: datasource=github-releases depName=open-policy-agent/conftest versioning=semver
ENV DEFAULT_CONFTEST_VERSION=0.35.0
```

In this case, we use GitHub releases on `open-policy-agent/conftest`, which means https://github.com/open-policy-agent/conftest/releases.

see also [supported datasource](https://docs.renovatebot.com/modules/datasource/#supported-datasources)